.env

```
@@ -41,6 +41,7 @@ COSMOS_DENOM=LIKE
 COSMOS_FRACTION_DENOM=nanolike
 COSMOS_FRACTION_DIGITS=9
 COSMOS_GAS_PRICE=0
+CURRENT_PREFIX=
```

I suggest naming starts with `COSMOS_` for Cosmos related config
```suggestion
COSMOS_ADDRESS_PREFIX=
```